docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -30,7 +30,15 @@ read or write (or no) access to project settings, logs, task
 annotations, and more. These can be requested for specific members or
 for entire guilds. The project admins (or Evergreen admins, if no
 project admins exist) are then able to approve/reject requested access
-for their project.
+for their project. 
+
+In order to access this functionality, you must look up the the MANA resource 
```

Comment:
Can we also make it clear that the website to start requesting is the MANA website, and not Evergreen (or alternatively, just link to the place in MANA to start searching for Evergreen projects)?